docs/standard/basetypes/commontypesystem.md

```diff
@@ -0,0 +1,446 @@
+---
+title: Common Type System
```

Comment:
We already have a common-type-system.md file in the standard folder.  Maybe we should combine the content?